### ISSUE TITLE
where to place FontAwesome file

### ISSUE THREAD

**Issue by lukegaylor** [posquit0/Awesome-CV#115]
So i running tex live on windoens and cannot for the life of me work out where to save the font file.

I have used the following question to try and guide me:
http://tex.stackexchange.com/questions/88423/manual-font-installation

Do I just need to install the .ttf files or the .sty files.
Currently I have all in ~texmf\tex\xelatex\fonts\truetype\awesomefont

And I am given the warning:
fontspec error: "font-not-found" The font FontAwesome cannot be found....

**Comment by johannesbottcher** [comment]
Why do you want to place the file anywhere? The fontawesome package is installed with a default TeX Live installation, so there shouldn't be any problem.

**Comment by lukegaylor** [comment]
Well i have re-installed Texlive 2016 this morning and I am still getting the same error, the first one appears on line 164 "fontspec error: "font-not-found" The font FontAwesome cannot be found...."
So my pdf generates with all the headings (Education, Skills, Experience...) but not the heading with name and contact details

**Comment by johannesbottcher** [comment]
Try the following example. If it doesn't run, ask a question on [TeX.SX](tex.tackexchange.com).
```syntaxhighlight=latex
\documentclass{article}
\usepackage{fontspec}
\RequirePackage{fontawesome}
\begin{document}
\faCodeFork
\end{document}
```

**Comment by lukegaylor** [comment]
The example ran and exited normally, but only generated a blank page. Was that the purpose? I only see the page number.

**Comment by johannesbottcher** [comment]
You should see the symbol for a code fork. I suggest to ask on TeX.SX as this is really irrelevant to the template. It is a general LaTeX issue.

**Comment by justinbass** [comment]
Check out my comment here, if it helps: https://github.com/posquit0/Awesome-CV/issues/147

**Comment by aptxna** [comment]
For this Font issue in MacOS, just copy the .ttf files in the fonts folder of Awesome-CV to the folder of system "/Library/Fonts/", and then I've solved this issue by doing so.

**Comment by 77QingLiu** [comment]
@aptxna where this the folder `/Library/Fonts/` located in?

**Comment by OJFord** [comment]
Closed via #344.
